examples/experimental/point-cloud-laz-vr/app.js

```diff
@@ -0,0 +1,245 @@
+/* global document, window,*/
```

Comment:
_navigator_ is only available in the browser context, need to add it here to make linter happy.
/* global document, window, navigator*/